### ISSUE TITLE
Turning Turrets on Air Units

### ISSUE THREAD

**Issue by Ineedause** [Anuken/Mindustry-Suggestions#2944]
### Describe what you would like changed, and why.

Some units like the Eclipse have turning turrets, these units also move slower when facing backwards. As of right now the following units will face the direction of the turrets when firing if controlled by a player:

1. Eclipse
2. Atumbruh
3. Zenith

When these units turn to face the direction of shooting it gives them some sort of disadvantage. The air units move slower if facing backwards.

### Describe the changes you want to propose. Include possible alternatives.

Change it so that the units body will still face your direction and only the turrets turn so that these disadvantages do not occur. Not only will this make more sense but it will lessen frustration. Other units already have this, for example the Beta or the Mega, there is no reason this should not apply to other units as well. 

The alternative, make it so that air units don't slow down when facing backwards.

**Comment by Ymabert** [comment]
@Ineedause yes, it would be a good thing, I also hope they fix it as soon as possible.

**Comment by oinkymoo** [comment]
the alpha and gamma also have this problem

**Comment by RealYeet** [comment]
> the alpha and gamma also have this problem

*This is my alt* The thing that I am trying to say is that some units that have turning turrets face the direction of fire. The alpha and gamma have fixed turrets that don’t turn so this does not apply to them

**Comment by oinkymoo** [comment]
ah gotcha
